### PR TITLE
Acquirer motors

### PR DESCRIPTION
**Pull request by PrathamRawat**

### CODE REVIEW

File: .classpath [hunk 1]
```diff
@@ -10,6 +10,5 @@
 	<classpathentry kind="var" path="USERLIBS_DIR/CTRE_Phoenix.jar"/>
 	<classpathentry kind="var" path="USERLIBS_DIR/CTRE_Phoenix-sources.jar"/>
 	<classpathentry kind="var" path="USERLIBS_DIR/navx_frc.jar"/>
-	<classpathentry kind="var" path="USERLIBS_DIR/CTRLib.jar"/>
```
Comment (adrisj7) [inline .classpath]: You should checkout .classpath, recommit and repush. This change should not be here

File: src/org/usfirst/frc/team694/robot/commands/AcquirerLoosenHoldCommand.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    public AcquirerLoosenHoldCommand() {
+        super();
+        requires(Robot.acquirer);
+        // Use requires() here to declare subsystem dependencies
+        // eg. requires(chassis);
+    }
+
+    // Called once when the command executes
+    protected void initialize() {
+        Robot.acquirer.releaseCube();
```
Comment (adrisj7) [inline src/org/usfirst/frc/team694/robot/commands/AcquirerLoosenHoldCommand.java]: While you're at it, I'd consider thinking about what comes to mind when you say "release". The acquirer can technically "release" the cube by spinning the wheels backwards to push the cube out. (Just a suggestion)

File: src/org/usfirst/frc/team694/robot/subsystems/Acquirer.java [hunk 3]
```diff
@@ -43,10 +48,19 @@ public void deacquire() {
...
 
     public void flipUp() {
-        acquirerSolenoid.set(true);
+        acquirerFlipSolenoid.set(true);
     }
 
     public void flipDown() {
-        acquirerSolenoid.set(false);
+        acquirerFlipSolenoid.set(false);
+        Timer.delay(0.5);
```
Comment (TacoTechnica) [inline src/org/usfirst/frc/team694/robot/subsystems/Acquirer.java]: I'd be careful using this. If this is the same as `Thread.sleep` then your bot will stop moving for 0.5 seconds.

I'd say don't do any timing in the subsystems. That's what command groups are for.

**Review by TacoTechnica — CHANGES_REQUESTED** [review]
Classpath issue

(A Minor naming issue)

Remove timing in the Acquirer's `flipDown()` method,

File: src/org/usfirst/frc/team694/robot/subsystems/Acquirer.java (hunk 3 above)
Comment (PrathamRawat) [reply src/org/usfirst/frc/team694/robot/subsystems/Acquirer.java]: Oh my, I think that is the delay thing which stops the whole robot.

**Review by TacoTechnica — APPROVED** [review]
Looks good, we'll have to deal with command merge problems though

**Pull request merged**